apps/expo-go/ios/Podfile.properties.json

```
@@ -0,0 +1,5 @@
+{
+  "expo.jsEngine": "hermes",
+  "newArchEnabled": "true",
+  "EX_DEV_CLIENT_NETWORK_INSPECTOR": "true"
```

```suggestion
  "newArchEnabled": "true"
```
`EX_DEV_CLIENT_NETWORK_INSPECTOR` is not used in expo-go.